### PR TITLE
Update ocs-api-overview.rst

### PR DESCRIPTION
**Pull request by bpcurse**
- Updated the displayed data to resemble the actual query results as of v17.0.2
- Added `GET /ocs/v1.php/cloud/users` paragraph
- Fixed some typos

### CODE REVIEW

File: developer_manual/client_apis/OCS/ocs-api-overview.rst [hunk 1]
```diff
@@ -114,8 +165,12 @@ Values of the theming app are exposed though the capabilities API, allowing clie
         <url>https://nextcloud.com</url>
         <slogan>A safe home for all your data</slogan>
         <color>#0082c9</color>
+	<color-text>#ffffff</color-text>
```
Comment (skjnldsv) [inline developer_manual/client_apis/OCS/ocs-api-overview.rst]: identation

File: developer_manual/client_apis/OCS/ocs-api-overview.rst [hunk 2]
```diff
@@ -86,11 +136,12 @@ Clients can obtain capabilities provided by the Nextcloud server and its apps vi
      </meta>
      <data>
       <version>
-       <major>12</major>
+       <major>17</major>
```
Comment (skjnldsv) [inline developer_manual/client_apis/OCS/ocs-api-overview.rst]: You used tabs in the example above (lines ~100), shall we use them here too?

File: developer_manual/client_apis/OCS/ocs-api-overview.rst (hunk 1 above)
Comment (bpcurse) [reply developer_manual/client_apis/OCS/ocs-api-overview.rst]: This was due to the tabs/spaces mix. Should be ok now.

File: developer_manual/client_apis/OCS/ocs-api-overview.rst (hunk 2 above)
Comment (bpcurse) [reply developer_manual/client_apis/OCS/ocs-api-overview.rst]: Sure, that makes sense. Thank you for the fast review!
I have changed all indentations in the document to tabs.

File: developer_manual/client_apis/OCS/ocs-api-overview.rst [hunk 3]
```diff
@@ -44,24 +44,74 @@ This request returns the available metadata of a user. Admin users can see the i
...
+			<backend>Database</database>
+			<subadmin/>
 			<quota>
-				<free>338696790016</free>
-				<used>7438874</used>
-				<total>338704228890</total>
-				<relative>0</relative>
-				<quota>-3</quota>
+				<free>20632824998</free>
+   				<used>842011482</used>
```
Comment (skjnldsv) [inline developer_manual/client_apis/OCS/ocs-api-overview.rst]: spaces :)

Comment (bpcurse) [reply developer_manual/client_apis/OCS/ocs-api-overview.rst]: Darn spaces ;)
I think now I've got them all in the API response blocks...

**Pull request merged**